spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/EventListenerMenas.scala

```diff
@@ -39,6 +45,15 @@ class EventListenerMenas(dao: MenasDAO,
   private var _runUniqueId: Option[String] = None
   private var _runNumber: Option[Int] = None
   private var _runStatus: RunStatus = RunStatus(RunState.running, None)
+  private var _controlMeasure: Option[ControlMeasure] = None
```

Comment:
Btw, why the vars' names start with underscore? I know this naming convintion is used to signal, they are private in languages lacking access modifiers. But in those, that have them?

Reply:
Not sure why these variable names start with an underscore. I just followed the convention so that at least the new code is consistent with the existing one.

Reply:
Fair enough